.gitignore

```diff
@@ -1,13 +1,14 @@
 *.pyc
-openedx_webhooks.db
-jira.pem
-jira.pub
-jira.uuid
-docs/_build
+.cache
 .coverage
-htmlcov
 .env
-.cache
+README.md
```

Comment:
Definitely the goal, but didn't want that to hold up what I need to focus on immediately.